### PR TITLE
Setup timer update

### PR DESCRIPTION
**Pull request by PiZZAD0X**
- Ports the setup timer to the new FW
- Updated dialog references to import
- Updated setup timer dialog (needs tweaking, redesign)
- Rewrote setup timer function

### CODE REVIEW

File: modules/modules.sqf [hunk 1]
```diff
@@ -53,7 +53,7 @@ Additional modules that can be enabled by removing the //
 //#include "rank_icons\root.sqf"
 //#include "remove_map\root.sqf"
 //#include "set_face\root.sqf"
-//#include "setup_timer\root.sqf"
+#include "setup_timer\root.sqf"
```
Comment (EM-Creations) [inline modules/modules.sqf:56]: Do we want to enable it by default?

File: modules/setup_timer/CfgNotifications.hpp [hunk 2]
```diff
@@ -1,9 +1,9 @@
-class SetupTimerEnded {
-	title = "SETUP TIMER";
-	iconPicture = "\a3\Ui_f\data\GUI\Cfg\Notifications\tridentEnemy_ca.paa";
-	description = "<t>Setup Timer Over, Fight!</t>";
-	color[] = {1,1,1,1};
-	colorIconPicture[] = {0.5,0,0,0.5};
-	colorIconText[] = {1,1,1,1};
-	duration = 5;
+class GVAR(TimerEnded) {
+    Title = "SETUP TIMER";
```
Comment (EM-Creations) [inline modules/setup_timer/CfgNotifications.hpp:2]: Does it matter that this field starts with upper case and the rest are lower case?

File: modules/setup_timer/functions/STIMER/fn_addSetupMarker.sqf [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+};
+if (_sides isEqualType sideUnknown) then {
+    if (_sides isEqualto sideUnknown) then {
+        _sides = [west, east, independent, civilian];
+    } else {
+        _sides = [_sides];
+    };
+};
+
+GVAR(SetupTimers) pushBackUnique [_area, _waitTime, _sides];
```
Comment (EM-Creations) [inline modules/setup_timer/functions/STIMER/fn_addSetupMarker.sqf:26]: Didn't know about `pushBackUnique` that's cool.

File: modules/setup_timer/CfgNotifications.hpp (hunk 2 above)
Comment (PiZZAD0X) [reply modules/setup_timer/CfgNotifications.hpp:2]: no, but pretty

File: modules/modules.sqf (hunk 1 above)
Comment (PiZZAD0X) [reply modules/modules.sqf:56]:
```suggestion
//#include "setup_timer\root.sqf"
```

**Review by EM-Creations — APPROVED** [review]

**Pull request merged**